convert2rhel/applock.py

```diff
@@ -59,15 +59,13 @@ class ApplicationLock:
     def __init__(self, name):
         # Our application name
         self._name = name
-        # Do we think we locked the pid file?
-        self._locked = False
         # Our process ID
         self._pid = os.getpid()
```

Comment:
```suggestion
        # Our process ID. We save this when the lock is created so it will be consistent even if we check from inside a fork.
        self._pid = os.getpid()
```